CMakeLists.txt

```diff
@@ -1,7 +1,14 @@
 cmake_minimum_required(VERSION 3.10)
 project(Wasmtime VERSION 0.27.0)
 
+option(CXX_STANDARD_20 "Use C++20" OFF)
```

Comment:
It is better to use something like this:
```cmake
if (NOT CMAKE_CXX_STANDARD)
  message(STATUS "Cannot detect C++ Standard. Switching to C++17 by default !!")
  set(CMAKE_CXX_STANDARD 17)
endif()
message(STATUS "CMAKE_CXX_STANDARD is ${CMAKE_CXX_STANDARD}")
set(WASMTIME_CPP_SUPPORTED_CXX_STANDARDS "17;20")
if (NOT CMAKE_CXX_STANDARD IN_LIST WASMTIME_CPP_SUPPORTED_CXX_STANDARDS)
  message(FATAL_ERROR "WASMTIME_CPP library does not support ${CMAKE_CXX_STANDARD}")
endif()
```

Reply:
I agree that's much better!

(I'm very much a cmake newb)